### DISCUSSION TITLE
Kaspersky Cyberthreat Map V0.1alpha Live Wallpaper

### DISCUSSION THREAD

**Discussion by ultropia** [rocksdanister/lively#637] [Show and tell]
Greetings again!
I made it, this morning...I will probably update more as ideas come flowing in.
![dentc2n-a70b7383-e06e-40d3-8328-711dc817e653](https://user-images.githubusercontent.com/87106735/126742512-1f4c2090-3848-4ab2-b268-02f73bd2f18e.png)

I have so many ideas!

[Kaspersky Cyberthreat Map V0.1alpha By Exokinetic
](https://www.deviantart.com/exokinetic/art/Kaspersky-Cyberthreat-Map-V0-1alpha-By-Exokinetic-886526303)

**Comment by erwinwallpaper** [comment]
WOW that is very cool.
